File: JJAndroidFramework/jj_android_service/src/main/java/com/example/hamilyjing/jj_android_service/TestService/Request/JJWeatherRequest.java
package com.example.hamilyjing.jj_android_service.TestService.Request;

import com.example.hamilyjing.jj_android_service.Tool.JJNetwork.JJRequest;

/**
 * Created by JJ on 4/13/16.
 */
public class JJWeatherRequest extends JJRequest {

    @Override
    public String baseUrl() {
        return "http://apis.baidu.com/showapi_open_bus/weather_showapi/areaid";
    }
}

File: JJiOSFramework/JJ_iOS_Service/JJ_iOS_Service/Src/TestService/Network/JJWeatherRequest.m
//
//  JJWeatherRequest.m
//  JJ_iOS_Service
//
//  Created by JJ on 4/14/16.
//  Copyright © 2016 JJ. All rights reserved.
//

#import "JJWeatherRequest.h"

@implementation JJWeatherRequest

- (instancetype)initWithOperationType:(NSString *)operationType_
                           parameters:(NSDictionary *)parameters_
                           modelClass:(Class)modelClass_
                       isSaveToMemory:(BOOL)isSaveToMemory_
                         isSaveToDisk:(BOOL)isSaveToDisk_
{
    self = [super initWithOperationType:operationType_ parameters:parameters_ modelClass:modelClass_ isSaveToMemory:isSaveToMemory_ isSaveToDisk:isSaveToDisk_];
    if (self)
    {
        self.userCacheDirectory = @"JJTestService";
    }
    
    return self;
}

- (NSString *)baseUrl
{
    return @"http://apis.baidu.com/showapi_open_bus/weather_showapi/areaid";
}

@end
